tasks/adventure_solutions/task_7_while.py

```diff
@@ -24,26 +24,28 @@ def dead(death_message):
 name = input("What is your name? ")
 print(f"Welcome to the dungeon, {name}!")
 
-print("Do you go through door 1 or door 2?")
-door = input("> ")
-
-while door != "1" and door != "2":
+door = 0
+while door not in ('1', '2'):
     wrong_input("1, 2")
-    print("Let's try again: do you go through door 1 or door 2?")
+    print("Do you go through door 1 or door 2?")
     door = input("> ")
 
-
 if door == "1":
     print(f"{door_greetings[door]}")
-    print(f"Look who's here! Your friend {friends[random.randint(0,4)]} :D")
+    rand_int = random.randint(0,4)
+    print(f"Look who's here! Your friend {friends[rand_int]} :D")
 
     print("There is a nice vampire asking you if you enjoy life.")
     print("What do you do?")
     print("A. Smile and nod")
     print("B. Scream and run")
 
     vampire = input("> ")
+<<<<<<< Updated upstream
     while vampire not in {"A", "B"}:  #  {"A", "B"} is a python set. Sets are great to check if a value is included
+=======
+    while vampire not in ('A', 'B'):
+>>>>>>> Stashed changes
```

Comment:
There are conflicts here. I think it is better to keep the set with the double quotes. 
Double quoting is used in all this file.